src/azure-cli/azure/cli/command_modules/eventhubs/tests/latest/test_eventhub_commands_privateendpoint_test.py

```diff
@@ -32,29 +32,29 @@ def test_eh_privateendpoint(self, resource_group):
         })
 
         # Prepare network
-        self.cmd('network vnet create -n {vnet} -g {rg} -l {loc} --subnet-name {subnet}',
+        self.cmd('network vnet create -n {vnet} -g test-migration -l {loc} --subnet-name {subnet}',
```

Comment:
Please don't use fixed resource for test, others can't rerun these tests later

Reply:
Resolved!

Reply:
Have you pushed your changes? What I can see is still using `test-migration` resource group prepared by yourself in advance

Reply:
I see, I was thinking about resources instead of resource-group.
Sorry for the inconvenience. Resolved!
Thank you!